parachains/runtimes/assets/statemint/src/lib.rs

```diff
@@ -531,12 +531,23 @@ impl pallet_collator_selection::Config for Runtime {
 	type WeightInfo = weights::pallet_collator_selection::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const UseUserConfiguration: bool = false;
+}
+
 impl pallet_asset_tx_payment::Config for Runtime {
 	type Fungibles = Assets;
 	type OnChargeAssetTransaction = pallet_asset_tx_payment::FungiblesAdapter<
 		pallet_assets::BalanceToAssetBalance<Balances, Runtime, ConvertInto>,
 		AssetsToBlockAuthor<Runtime>,
 	>;
+	type WeightInfo = ();
```

Comment:
is this supposed to be without weights?

Reply:
we do not use statemine. we do not host statemine. 
so will generate weights only in our runtime.

Reply:
we do not host in node, but use in rust xcmp tests as some parachain, so it should be compilable. but do not care wights in such tests.

Reply:
so for sure parity will not host it without weighs, but we do not host that stuff.